nrfx/mdk/nrf5340_network_bitfields.h

```diff
@@ -8071,15 +7789,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #define RESET_RESETREAS_RESETPIN_NotDetected (0UL) /*!< Not detected */
 #define RESET_RESETREAS_RESETPIN_Detected (1UL) /*!< Detected */
 
-/* Register: RESET_NETWORK_FORCEOFF */
-/* Description: Force off power and clock in network core */
-
-/* Bit 0 : Force off power and clock in network core */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Pos (0UL) /*!< Position of FORCEOFF field. */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Msk (0x1UL << RESET_NETWORK_FORCEOFF_FORCEOFF_Pos) /*!< Bit mask of FORCEOFF field. */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Release (0UL) /*!< Release force off signal */
-#define RESET_NETWORK_FORCEOFF_FORCEOFF_Hold (1UL) /*!< Hold force off signal */
```

Comment:
This needs a corresponding update (adding of `#if defined ...`) in `hal/nrf_reset.h`.